### PR TITLE
Fix collapsed menu

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following:** 

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

Fix text being shown despite side-menu collapsed

**How does this PR accomplish the above?:**

Re-add incorrectly removed `<span>` elements

Screenshot from 2021-12-31 17-35-29

**What documentation changes (if any) are needed to support this PR?:**

None

### CODE REVIEW

File: scripts/pi-hole/php/header.php [hunk 1]
```diff
@@ -416,80 +416,80 @@ function pidofFTL()
...
+                    <i class="fa fa-fw menu-icon fa-clock"></i> <span>Long-term data</span>
                     <span class="pull-right-container">
                       <i class="fa fa-angle-left pull-right"></i>
                     </span>
                   </a>
                   <ul class="treeview-menu">
                     <li<?php if($scriptname === "db_graph.php"){ ?> class="active"<?php } ?>>
                         <a href="db_graph.php">
-                            <i class="fa fa-fw menu-icon fa-file-alt"></i> Graphics
+                            <i class="fa fa-fw menu-icon fa-file-alt"></i> <span>Graphics</span>
```
Comment (rdwebdesign) [inline scripts/pi-hole/php/header.php:439]: **Note 1:**
The `<span>` tags are necessary only for main items (ex: line 431).
They are not necessary for sub-items, inside `<ul class="treeview-menu">` (but it's not a problem).

**Note 2:** 
accidentally, I added this tags to #2040.

**Comment by yubiuser** [comment]
Merge conflicts ahead

File: scripts/pi-hole/php/header.php (hunk 1 above)
Comment (DL6ER) [reply scripts/pi-hole/php/header.php:439]: I added it for all of them intentionally to hopefully avoid forgetting about them again in the future. How does one *accidentally* do this?

**Comment by DL6ER** [comment]
Seems it's not needed anymore

File: scripts/pi-hole/php/header.php (hunk 1 above)
Comment (rdwebdesign) [reply scripts/pi-hole/php/header.php:439]: > How does one _accidentally_ do this?

I was testing the other PR and saw the strange menu. I tried to fix it and forgot to revert the changes before send the PR.

**Pull request closed without merging**